### ISSUE TITLE
[feature-request]: Add aria-label to search icon in the navbar to make our websites more accessible

### ISSUE THREAD

**Issue by signekb** [seedcase-project/seedcase-website#260]
### What feature would you like to request?

Currently, the search icon doesn't have an aria-label, which makes it difficult for screen readers to locate it. To increase accessibility, we should investigate how to add an aria-label to the icon using Quarto. 

I haven't been able to find a way using Quarto (but it can maybe be done by overwritting the default HTML or adding a JS script), which is why I have created an issue in their repo [here](https://github.com/quarto-dev/quarto/issues/468).

EDIT: Found an issue in their quarto-cli repo which my issue seems to be a duplicate of. I have asked for a status on the issue on that issue as well. https://github.com/quarto-dev/quarto-cli/issues/678
